source/octf/proto/trace.proto

```diff
@@ -34,13 +34,14 @@ message EventDeviceDescription {
 
 /** @brief IO operation direction */
 enum IoType {
-    Read = 0;
-    Write = 1;
-    Discard = 2;
+    UnknownIoType = 0;
+    Read = 1;
+    Write = 2;
+    Discard = 3;
 }
 
 enum FsEventType {
-    Unknown = 0;
+    UnknownFsType = 0;
```

Comment:
I think Unknown is enough

Reply:
Enum elements have proto file scope (not enum scope). So if we define Unknown two times, protoc throws an error.

Reply:
You are right But rename it to UnknownFsEventType